### PR TITLE
Fix accelerator key clashes

### PR DESCRIPTION
**Pull request by softins**
**Short description of changes**

Corrected accelerator key clashes in English and several other languages.

**Context: Fixes an issue?**

Part of release process

**Does this change need documentation? What needs to be documented and how?**

Probably not. Accelerator keys vary between languages and are shown in the GUI

**Status of this Pull Request**

Ready for review

**What is missing until this pull request can be merged?**

~Sanity test on my system~ DONE

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I tested my code and it does what I want
- [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#source-code-consistency) 
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I've filled all the content above

### CODE REVIEW

**Review by pljones — APPROVED** [review]

**Comment by softins** [comment]
Having edited the ts files by hand and committed, I did an `lupdate` and verified that it didn't change them.

**Review by hoffie — APPROVED** [review]
Looks good to me, but can't test myself at the moment.

**Review by ann0see — COMMENTED** [review]
The Verify Translations workflow didn’t run on this PR (although src/res/translation) was changed. Probably there’s a typo somewhere in my workflow?

**Comment by pljones** [comment]
Hm.  Can't check right now - you'll probably have found it before I can! :)

**Comment by ann0see** [comment]
Maybe a "*" is missing

File: src/res/translation/translation_fr_FR.ts [hunk 1]
```diff
@@ -738,7 +738,7 @@
     <message>
         <location filename="../../clientdlg.cpp" line="376"/>
         <source>C&amp;hat...</source>
-        <translation>&amp;Tchate...</translation>
+        <translation>Tc&amp;hate...</translation>
```
Comment (jujudusud) [inline src/res/translation/translation_fr_FR.ts:741]: For french accelerators keys, "T" is a good key for "Tchate".
Please let it as it is.

Comment (softins) [reply src/res/translation/translation_fr_FR.ts:741]: There is another item in that menu with T as the accelerator. One of them needed to be changed.

Comment (jujudusud) [reply src/res/translation/translation_fr_FR.ts:741]: Could you please tell me which one is the other ? I can't locate it.

Comment (softins) [reply src/res/translation/translation_fr_FR.ts:741]: Pas de &tri des canaux

Comment (jujudusud) [reply src/res/translation/translation_fr_FR.ts:741]: I think it is not the same context. I will have a look to all the accelerator key for the next release as soon as possible.

Comment (softins) [reply src/res/translation/translation_fr_FR.ts:741]: From my 3.8.2:
image

Comment (softins) [reply src/res/translation/translation_fr_FR.ts:741]: I guess an alternative could be `&Pas de tri des canaux` allowing `&Tchate` within the menu, as that doesn't conflict with `&Tchate` in the main dialog.

Chat within the View menu was a new addition after 3.8.1

Comment (pljones) [reply src/res/translation/translation_fr_FR.ts:741]: Make "&P" for _premier_ (as that's what the option does); make "&C" for _canaux_ (as that's what the option does); make "&T" for _Tchate_ (as that's what the option does).  That way they're mnemonic.

**Pull request merged**